pkg/virt-api/rest/subresource.go

```diff
@@ -29,6 +29,8 @@ import (
 	"strings"
 	"sync"
 
+	k6types "kubevirt.io/kubevirt/pkg/util/types"
```

Comment:
```suggestion
	utiltypes "kubevirt.io/kubevirt/pkg/util/types"
```
Also could you group with other kubevirt dependecies?

Reply:
I am forced to ask - why `k6types` ? Is there a reason? Was this a typo ?

Reply:
Not sure why you're so surprised TBH. `k6t` is used in many places in the codebase already [[1](https://github.com/kubevirt/kubevirt/blob/main/pkg/monitoring/domainstats/collector.go#L26)][[2](https://github.com/kubevirt/kubevirt/blob/main/pkg/monitoring/domainstats/prometheus/prometheus.go#L35)][[3](https://github.com/kubevirt/kubevirt/blob/main/pkg/network/setup/podnic.go#L361)]. I think it's pretty cool, short and clean.

But there was a missing `t` over there. Should be `k6ttypes`.

Reply:
> Also could you group with other kubevirt dependecies?

Yes, will do

Reply:
But what does it mean ? Honestly, I have no clue. 

It may be clean - not sure about cool - and is short. But I'm clueless as to what it is about. That's a bad thing for an import, isn't it ?

Reply:
It took me also a couple of minutes :) the first time I saw this. It is like k8s as the number of letters. K6 = k- U-B-V-I-R-T

Reply:
@maiqueb tbh, I think we should improve the nomenclature of importing kubevirt packages independently to this PR. We have a mixture of `k6t`, `kvirt` or only  the version`v1` (that confuses them with the k8s package)

Reply:
I think we shouldn't use `k6ttypes` now. In other places we use `utiltypes` so why do we want to break this convention here. Also if we decide to replace the naming to `k6t...` then it will be easier to do with search-and-replace.

Reply:
I think we should have a general discussion on conventions for package names (among other conventions). This could be a great topic to bring up for SIG code-quality.

My vote would be in favor of k6t, for the same reasons k8s chose this path. It's much more compact than something like `kubevirtutils`. In any case, since it's being used in many places throughout our codebase, I agree with @alicefr that we should discuss this on a broader scope and agree on a convention (and perhaps even enforce it).